tests/integration/test_auditlogs.py

```diff
@@ -1,16 +1,44 @@
 from datetime import datetime
 from datetime import timedelta
+from shlex import split as split_command
 
 import pytest
+from tests.integration.util import assert_test
+from tests.integration.util import DataServer
 
-SEARCH_COMMAND = "code42 audit-logs search"
+from code42cli.main import cli
+
+
+SEARCH_COMMAND = "audit-logs search"
 BASE_COMMAND = "{} -b".format(SEARCH_COMMAND)
-begin_date = datetime.utcnow() - timedelta(days=-10)
+begin_date = datetime.utcnow() - timedelta(days=2)
 begin_date_str = begin_date.strftime("%Y-%m-%d %H:%M:%S")
-end_date = datetime.utcnow() - timedelta(days=10)
+end_date = datetime.utcnow() - timedelta(days=0)
 end_date_str = end_date.strftime("%Y-%m-%d %H:%M:%S")
 
 
+@pytest.mark.integration
+@pytest.mark.parametrize(
+    "command,protocol",
+    [
+        (
+            "audit-logs send-to localhost:5140 -p TCP -b '{}'".format(begin_date_str),
+            "TCP",
+        ),
+        (
+            "audit-logs send-to localhost:5140 -p UDP -b '{}'".format(begin_date_str),
+            "UDP",
+        ),
+    ],
+)
+def test_auditlogs_send_to(runner, integration_test_profile, command, protocol):
+    with DataServer(protocol=protocol):
+        result = runner.invoke(
+            cli, split_command(command), obj=integration_test_profile
+        )
+    assert result.exit_code == 0
+
+
 @pytest.mark.integration
 @pytest.mark.parametrize(
```

Comment:
We should split this up into individual tests

Reply:
resolving this conversation as it is addressed [here](https://github.com/code42/code42cli/pull/212#discussion_r567613905)